scripts/run_upgrade.sh

```diff
@@ -14,8 +14,7 @@ done
 echo "Installing Workflow chart from the '${ORIGIN_WORKFLOW_REPO}' chart repo..."
 # shellcheck disable=SC2046
 helm install "${ORIGIN_WORKFLOW_REPO}"/workflow --namespace=deis \
-  $(set-chart-version workflow) $(set-chart-values workflow) --set controller.registration_mode=enabled
-# TODO: remove this "registration_mode" override when e2e tests expect "admin_only" as the default
+  $(set-chart-version workflow) $(set-chart-values workflow)
```

Comment:
It was intentional... since the value must be set on upgrade, it doesn't need to be set on initial install.  (value only necessary for post-upgrade e2e run)